docs/docsite/rst/playbook_guide/playbooks_filters.rst

```diff
@@ -27,7 +27,7 @@ You can provide default values for variables directly in your templates using th
 
     {{ some_variable | default(5) }}
 
-In the above example, if the variable 'some_variable' is not defined, Ansible uses the default value 5, rather than raising an "undefined variable" error and failing. If you are working within a role, you can also add a ``defaults/main.yml`` to define the default values for variables in your role.
+In the above example, if the variable 'some_variable' is not defined, Ansible uses the default value 5, rather than raising an "undefined variable" error and failing. If you are working within a role, you can also add a ``defaults/main.yml`` or ``defaults/main`` directory to define the default values for variables in your role.
```

Comment:
i would just rephrase to add 'role defaults' and have the full explanation in the role docs (and link to that)

Reply:
which i see you do below, just add an anchor